tests/B2CWebAppCallsWebApi/Client/Controllers/TodoListController.cs

```diff
@@ -4,34 +4,38 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Identity.Web;
+using System.Collections.Generic;
+using System.Net.Http;
 using System.Threading.Tasks;
-using TodoListClient.Services;
 using TodoListService.Models;
 
 namespace TodoListClient.Controllers
 {
     public class TodoListController : Controller
     {
-        private readonly ITodoListService _todoListService;
         private readonly ITokenAcquisition _tokenAcquisition;
+        private readonly IDownstreamWebApi _downstreamWebApi;
```

Comment:
thanks for doing all this! this is so much better. we need to update the sample on this release as well.